### ISSUE TITLE
User Management

### ISSUE THREAD

**Issue by novalagung** [eaciit/mq#2]
Module to manage users. Accessible on [http://localhost:1234/user](http://localhost:1234/user). There are two section. 
### Grid User
- The first is for show all data in grid. This grid have three columns, `Username`, `Role` and `Options` (which contain two buttons `edit` and `delete`). 
- There is also a `filter` which can be seen above the grid. We can type anything and then press enter or click `search` button to make the grid contents being filtered.
- Next to it, there is `create new user` button. Section _Form User_ will showing up when this button clicked.
- `Edit` button on each row on the grid is used to make changes for selected user. it' open section \* _Form User_ (same like `create new user` button), and fill `username` and `role` with selected data. 
- `Delete` button is for deleting selected row.

Preview:

![screen shot 2015-04-27 at 6 02 10 pm](https://cloud.githubusercontent.com/assets/982868/7346371/cbe6d2ae-ed07-11e4-983d-8579547d36ca.png)

---
### Form User

Next is section `Form User`. This section is for inserting or editing data. 
- On upside, there are three buttons. `Back to list` for going back to the grid. `Reset` button to clear all filled value in inputs. and `save` button to save changes 
- There are 4 inputs in this section, `username`, `password`, `password confirmation`, and `role`.
- All input has validations (_not empty_, _minimum character_, _password confirmation_)
- When in edit mode, username and role will be disabled, currently only _change password_ feature are supported in edit mode.

![screen shot 2015-04-27 at 6 02 16 pm](https://cloud.githubusercontent.com/assets/982868/7346436/629c316c-ed08-11e4-9e92-ae4140ca9703.png)

---
### TL;DR

All functionality is working smoothly. currently on edit section, only change password supported.
